plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxConnectorTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import io.trino.testing.QueryRunner;
+
+import static io.trino.plugin.influxdb.InfluxQueryRunner.createInfluxQueryRunner;
+
+public class TestInfluxConnectorTest
+        extends BaseInfluxConnectorTest
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        TestingInfluxServer server = closeAfterClass(new TestingInfluxServer("1.4.0"));
```

Comment:
Do we need to test it for multiple version or can we start testing from 1.80 ?

Reply:
in influxdb.rst i wrote supported from 1.40 ~ 1.8.10 , so i think if oldest version test is neccessary.

Reply:
But is there is any specific reason for supporting 1.40 ? I see the latest stable version is 2.6 and can we add support from this version directly ? This is the first version of the connector and it doesn't make much sense to test it on a older version unless it is some sort of LTS supported by InfluxDB. WDYT ?

Reply:
Emm，yea，i thought you said make sense，i will change tests and rst to version 1.8 support s. 1.8.10 is the latest open source version for influxdb and v2.x is incompatible. So we begin from v1.8.0 to 1.8.10，that's okay?

Reply:
Do we need new client for 2.x version ?

Reply:
> Do we need new client for 2.x version ?

yes, i want to do that in next pr. v2.x is not open source and is not used as widely as v1.x.